<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>355</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>easyQR</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #211D1F;
color: #FDFBF7;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_generate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid rgba(255, 255, 255, 0.05);
  border-radius: 8px;
  padding: 4px 6px;
  background-color: #2C282B;
  color: #FDFBF7;
}
QPushButton:hover {
  background-color: #272325;
}
</string>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_result">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 16px;
padding: 16px;
background-color: white;
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_data">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid rgba(255, 255, 255, 0.05);
border-radius: 8px;
padding: 4px 6px;
selection-background-color: rgba(46, 204, 113, 0.2);
selection-color: #FDFBF7;</string>
    </property>
    <property name="placeholderText">
     <string>Введите текст для кодирования</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ecc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите уровень коррекции ошибок</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_">
     <item>
      <widget class="QRadioButton" name="radioButton_ecc_L">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
  color: #8f8c8b;
}
QRadioButton::indicator {
  width: 8px;
  height: 8px;
  background-color: rgba(255, 255, 255, 0.2);
  border: 4px solid rgba(255, 255, 255, 0.2);
  border-radius: 8px;
  margin-right: 2px;
}
QRadioButton::indicator:checked {
  border-color: #2ecc71;
}</string>
       </property>
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ecc_M">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
  color: #8f8c8b;
}
QRadioButton::indicator {
  width: 8px;
  height: 8px;
  background-color: rgba(255, 255, 255, 0.2);
  border: 4px solid rgba(255, 255, 255, 0.2);
  border-radius: 8px;
  margin-right: 2px;
}
QRadioButton::indicator:checked {
  border-color: #2ecc71;
}</string>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ecc_Q">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
  color: #8f8c8b;
}
QRadioButton::indicator {
  width: 8px;
  height: 8px;
  background-color: rgba(255, 255, 255, 0.2);
  border: 4px solid rgba(255, 255, 255, 0.2);
  border-radius: 8px;
  margin-right: 2px;
}
QRadioButton::indicator:checked {
  border-color: #2ecc71;
}</string>
       </property>
       <property name="text">
        <string>Q</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ecc_H">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton {
  color: #8f8c8b;
}
QRadioButton::indicator {
  width: 8px;
  height: 8px;
  background-color: rgba(255, 255, 255, 0.2);
  border: 4px solid rgba(255, 255, 255, 0.2);
  border-radius: 8px;
  margin-right: 2px;
}
QRadioButton::indicator:checked {
  border-color: #2ecc71;
}</string>
       </property>
       <property name="text">
        <string>H</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid rgba(255, 255, 255, 0.05);
  border-radius: 8px;
  padding: 4px 6px;
  background-color: #2C282B;
  color: #FDFBF7;
}
QPushButton:hover {
  background-color: #272325;
}
</string>
    </property>
    <property name="text">
     <string>Сохранить в PNG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: 1px solid rgba(255, 255, 255, 0.05);
  border-radius: 8px;
  padding: 4px 6px;
  background-color: #2C282B;
  color: #FDFBF7;
}
QPushButton:hover {
  background-color: #272325;
}
</string>
    </property>
    <property name="text">
     <string>Скопировать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Генератор QR-кодов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите цвет модулей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  border: none;
  background-color: #000;
}
QPushButton:pressed {
  border: none;
  padding: 0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_value">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  color: #8f8c8b;
}</string>
    </property>
    <property name="text">
     <string>#000000</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
